tsconfig.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Feels repetitive having a tsconfig in the root and then again in the packages folder?

Reply:
You are right, the tsconfig.base.json is overkill. I'll pull that config back into the root tsconfig.json. We need them in each package though, to manage the various inter-package dependency detection for intellisense .